src/main/controllers/helpers/AdminNotificationHelper.ts

```diff
@@ -17,7 +17,7 @@ export const getApplicationsWithTribunalOrderOrRequest = (
     for (const app of apps) {
       const allRequestsFromAdmin: AdminNotifcation[] = getVisibleRequestFromAdmin(app, translations, languageParam);
       if (!allRequestsFromAdmin.length) {
-        return appsWithTribunalOrderOrRequest;
+        continue;
```

Comment:
If one application didn't have requests from the admin it would stop getting further applications/requests

Reply:
you can just do if (allRequestsFromAdmin.length( for ...)

Reply:
Good spot. Updated.